### ISSUE TITLE
Idea: print the cause of exceptions in spdxToOsv()

### ISSUE THREAD

**Issue by esnible** [spdx/spdx-to-osv#9]
It is quite frustrating to see output like:

```
Error converting SPDX file to OSV.
Error reading the SPDX input file
```

I rebuilt, adding to Main.java
```Java
            if (Objects.nonNull(ex.getCause())) {
                System.err.println(ex.getCause());
            }
```

and could then see that the error was `Output file OSVOutput.json already exists.`.

**Comment by goneall** [comment]
@esnible - Good point - please go ahead and create a PR - I'm planning on producing a release with updated libraries in the next few days.

**Comment by goneall** [comment]
Resolved with PR #12
